lib/include/Spix/AnyRpcServer.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <Spix/TestServer.h>
+#include "spix_export.h"
```

Comment:
Actually prefere your ordering over the one of clang-format. Could you add an empty line above this include to make clang-format happy?

```suggestion

#include "spix_export.h"
```